docs/design-documents/features/storage/Configuration/CONFIGURATION.md

```diff
@@ -0,0 +1,388 @@
+# Storage configuration 
```

Comment:
design docs - were they reviewed by docs team or we request it now here? Let us know, we will add docs reviewers

Reply:
We had an internal review, but not by the docs team. Please add them.